app/templates/admin/manageDepartments.html

```diff
@@ -93,4 +94,43 @@ <h1>Manage Departments</h1>
         <div class="col-md-2"></div>
     </div>
 </div>
+<div class="modal" id="manageDepartmentSupervisorModal" tabindex="-1" role="dialogue" >
+  <div class="modal-dialog" role="document">
+    
+    
+    <div id="manageSupervisorContent" class="modal-content">
+      
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <div class="department-name"></div>
+        
+      </div>
+
+      <div class="modal-body ">
```

Comment:
Remove the trailing space character from your class string. I can't belive you left that in there